code/__DEFINES/inventory.dm

```diff
@@ -184,7 +186,8 @@ GLOBAL_LIST_INIT(detective_vest_allowed, typecacheof(list(
 	/obj/item/restraints/handcuffs,
 	/obj/item/storage/fancy/cigarettes,
 	/obj/item/tank/internals/emergency_oxygen,
-	/obj/item/tank/internals/plasmaman)))
+	/obj/item/tank/internals/plasmaman)) - typecacheof(list(
```

Comment:
why not just remove /obj/item/melee/baton ?

Reply:
Because that would remove them holding stunbatons

Reply:
just do blacklist stunprod

Reply:
sounds like exactly what i'm doin!

Reply:
No idea if this would work but it looks like you can run procs through it.
`proc/blacklist(list)`
`     var/blacklist = typecacheof(list(/obj/item/melee/baton/cattleprod))`
`     list -= blacklist`
`     return list`

then ``GLOBAL_LIST_INIT(security_hardsuit_allowed, blacklist(typecacheof(list(/obj/item/ammo_box ...)``

It's really anyone's guess as to whether or not it's more efficient to run the proc in a proc